.github/workflows/llvm-ci.yml

```diff
@@ -0,0 +1,263 @@
+        run: |
+          $ErrorActionPreference = 'Stop'
+          Write-Host "Using LLVM_TAG=$env:LLVM_TAG"
+
+          $vsPath = "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\Common7\Tools\VsDevCmd.bat"
+          cmd /c "`"$vsPath`" && set" | ForEach-Object {
+              if ($_ -match '^([^=]+)=(.*)') {
+                  [Environment]::SetEnvironmentVariable($matches[1], $matches[2])
+              }
+          }
+
+          New-Item -ItemType Directory -Force -Path llvm-project | Out-Null
+          Set-Location llvm-project
+          git init
+          git remote add origin https://github.com/llvm/llvm-project.git
+          git fetch --depth 1 origin "$env:LLVM_TAG"
+          git checkout --detach FETCH_HEAD
+
+          cmake -S llvm -B build -G "Visual Studio 17 2022" -A x64 `
+            -DCMAKE_BUILD_TYPE=Release `
+            -DCMAKE_INSTALL_PREFIX="$env:CMAKE_INSTALL_PREFIX" `
+            -DLLVM_ENABLE_PROJECTS="clang;lld" `
+            -DLLVM_ENABLE_RTTI:BOOL=ON `
+            -DLLVM_ENABLE_LIBXML2=OFF `
+            -DLLVM_ENABLE_ZLIB=OFF `
+            -DLLVM_ENABLE_ZSTD=OFF `
+            -DLLVM_TARGETS_TO_BUILD="host;NVPTX;AMDGPU" `
+            -DLLVM_ENABLE_TERMINFO=OFF `
+            -DCMAKE_MSVC_RUNTIME_LIBRARY="MultiThreadedDLL"
+
+          cmake --build build --config Release --target ALL_BUILD
+          cmake --build build --config Release --target INSTALL
+
+      # ----------  compute package naming ----------
+      - name: Compute package name
+        id: pkgmeta
+        shell: bash
+        run: |
+          set -euo pipefail
+          os="${RUNNER_OS}"
+          arch="$(uname -m || true)"
+          case "$os" in
+            Linux)
+              if [ "$arch" = "aarch64" ] || [ "$arch" = "arm64" ]; then suffix="linux-aarch64"; else suffix="linux-x86_64"; fi
+              ;;
+            macOS)  suffix="macos-arm64" ;; # macos-15 runners are Apple Silicon
+            Windows) suffix="windows-amd64" ;;
+          esac
+          echo "suffix=$suffix" >> "$GITHUB_OUTPUT"
+          echo "zip=taichi-llvm-${LLVM_VERSION}-$suffix.zip" >> "$GITHUB_OUTPUT"
+
+      # ----------  package (Linux/macOS) ----------
+      - name: Package (Linux/macOS)
+        if: runner.os != 'Windows'
+        env:
+          CMAKE_INSTALL_PREFIX: ${{ steps.prefix.outputs.prefix }}
+        run: |
+          set -euo pipefail
+          pkgroot="${RUNNER_TEMP}/taichi-llvm-${LLVM_VERSION}-${{ steps.pkgmeta.outputs.suffix }}"
+          mkdir -p "$pkgroot"
+          # copy install tree
+          cp -R "${CMAKE_INSTALL_PREFIX}/." "$pkgroot/"
+          # bundle license
+          cp llvm-project/LICENSE.TXT "$pkgroot/LLVM-LICENSE.txt" || true
+          # optional strip to shrink size
+          if [ -x "${CMAKE_INSTALL_PREFIX}/bin/llvm-strip" ]; then
+            find "$pkgroot/bin" -type f -exec "${CMAKE_INSTALL_PREFIX}/bin/llvm-strip" --strip-unneeded {} + || true
```

Comment:
why `|| true` ?

Reply:
(It's possible that the strip also fails on ubuntu x86 tbh, since that build didnt finish yet)

Reply:
@johnnynunez thoughts on this llvm-strip error on arm?